### DISCUSSION TITLE
Extending colors not working

### DISCUSSION THREAD

**Discussion by erfurtjohn** [tailwindlabs/tailwindcss#11783] [Help]
Hello.

I'm running AdonisJS with tailwindcss (followed installation instructions in the docs) and everything works fine, excepts the fact that extended colors won't work.

This is my configuration file:
```
/**` @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./resources/**/*.{edge,js,ts,jsx,tsx,vue}'],
  theme: {
    extend: {
      colors: {
        dwdorange: '#FB8C00',
        dwdred: '#E53935',
        dwdpurple: '#880E4F',
        dwdindexone: '#FFFFCD',
        dwdindextwo: '#FFD879',
        dwdindexthree: '#FF8C39',
        dwdindexfour: '#E9161D',
        dwdindexfive: '#7F0126',
      },
    },
  },
  plugins: [require('@tailwindcss/forms')],
}
```

Did I do something wrong?
I've already tried to remove the config file and reinstalled tailwindcss.

**Comment by wongjn** [comment]
The configuration seems fine. Consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Comment by erfurtjohn** [comment]
I did a further investigation and it I found missing files in the config...postcss simply ignored it. Thanks for the help anyway!
